instructions/section-4/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My portfolio</title>
+</head>
+<body>
+
+<section>
+Hello, I'm Maryna Radchenko! I'm passionate about software development and I'm skilled in HTML, JavaScripr, and React.
+</section>
```

Comment:
I'm not an expert on the `<section>` tag, but based on this [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) blurb:

> The `<section>` [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element represents a generic standalone section of a document, which doesn't have a more specific semantic element to represent it.

I think it makes sense for an about, experience, and contact section, but I think this should just be a `<p>` tag. Just something to consider. :smile: